test/options/form.test.js

```diff
@@ -0,0 +1,75 @@
+const assert = require('chai').assert;
+const { Builder, By, until } = require('selenium-webdriver');
+const test = require('selenium-webdriver/testing');
+
+test.describe('Options select form', function() {
+  this.timeout(50000);
+
+  const addr = 'http://localhost:3000';
+  let driver;
+
+  test.beforeEach(() => {
+    driver = new Builder()
+      .forBrowser('chrome')
+      .build();
+  });
+
+  test.afterEach(() => {
+    driver.quit();
+  });
+
+  test.xit('Loads', () => {
+    login(driver);
+    driver.findElement(By.id('logout'))
+      .getText()
+      .then(text => {
+        assert.equal(text, 'Log Out');
+      });
+  });
+
```

Comment:
@neight-allen Do these tests look OK to you? Under the 'Has an attractive UI' it block. I was hoping for some Capybara precision, but the tests I'm writing feel clunky. Maybe that's what Selenium tests feel like?